builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -271,7 +271,7 @@ node() {
 							endpointUrl = "https://${apiHostName}/${current_environment}/${environment_logical_id}/${config['domain']}/${config['service']}"
 						}
 
-						def apiArns = getAPIGatewayResourceARNs(aws_api_id, environment_logical_id)
+						def apiArns = getAPIGatewayResourceARNs(aws_api_id, current_environment)
```

Comment:
I think here "environment_logical_id" is needed. As current_environment values will be prod/dev. Please ensure this